nextflow.config

```diff
@@ -0,0 +1,31 @@
+manifest {
+    name = 'ERVmap'
+    mainScript = 'main.nf'
+}
+
+docker {
+    enabled = true
+    temp = 'auto'
+    runOptions = '--rm -u $(id -u):$(id -g)'
+}
+
+params {
+    inputDir=''
+    outPrefix='results/test.'
+    cpus=1
+    limitMemory=30000000
+    debug='off'
+}
+
+process {
+    withName: STARAlignment {
+        container = 'eipm/ervmap:latest'
+        containerOptions = '--memory 50G --memory-swap 100G -v /path/to/genome:/genome:ro'
+    }
+
+    withName: ERVcounting2 {
+        container = 'eipm/ervmap:latest'
+        containerOptions = ' -v $(pwd)/data:/data:ro -v /path/to/genome:/genome:ro' 
```

Comment:
I think we can add path to genome as a parameter and mount via the main.nf file instead of including in containerOptions.

Reply:
I thought about it. I ended up deciding to including it in the config as part of the more *static* options, i.e. immutable. I would have included in the docker image but I think it makes it too big as it's 28GB.